protostar/commands/test/environments/fuzz_test_execution_environment.py

```diff
@@ -43,10 +44,24 @@ def is_fuzz_test(function_name: str, state: TestExecutionState) -> bool:
     return bool(params)
 
 
+@dataclass
+class FuzzTestExecutionResult(TestExecutionResult):
+    fuzz_runs_count: int
+
+
 class FuzzTestExecutionEnvironment(TestExecutionEnvironment):
+    @dataclass
+    class FuzzConfig:
+        max_fuzz_examples: int = 100
```

Comment:
could this class be moved out to the top of this module, so that the stance `FuzzTestExecutionEnvironment.FuzzConfig` could be simplified to just `FuzzConfig`? without repetitive `Fuzz` part